onnx/shape_inference/implementation.h

```diff
@@ -77,22 +80,26 @@ struct GraphInferenceContext {
       const std::unordered_map<std::string, TypeProto*>&
           outer_scope_value_types_by_name_in,
       const std::unordered_map<std::string, int> opset_imports_in,
-      SymbolTable* symbolTable_in,
+      SymbolTable* symbol_table_in = nullptr,
       const ISchemaRegistry* schema_registry_in = OpSchemaRegistry::Instance(),
-      const int ir_version_in = IR_VERSION)
+      const int ir_version_in = IR_VERSION,
+      const ModelLocalFunctionsMap& model_local_functions_in = {},
+      GetFunctionMapIdFn get_func_id = nullptr)
       : outer_scope_value_types_by_name{&outer_scope_value_types_by_name_in},
         opset_imports{opset_imports_in},
+        symbol_table{symbol_table_in},
         schema_registry{schema_registry_in},
-        symbolTable{symbolTable_in},
-        ir_version{ir_version_in} {}
-
+        ir_version{ ir_version_in },
```

Comment:
tiny nit: sync space
```suggestion
        ir_version{ir_version_in},
```